src/mscorlib/shared/System/IO/FileStream.Unix.cs

```diff
@@ -166,6 +166,11 @@ private static Interop.Sys.OpenFlags PreOpenConfigurationFromOptions(FileMode mo
                     break;
             }
 
+            if ((share & FileShare.Inheritable) == 0)
```

Comment:
Nit: can you add a comment that we handle Inheritable here, but the rest of FileShare is handled later by Init, e.g. https://github.com/tmds/coreclr/blob/ea9a245d01bb1b8e9cb9935f7f792c58b4572ecc/src/mscorlib/shared/System/IO/FileStream.Unix.cs#L73